<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plugins</class>
 <widget class="QWidget" name="Plugins">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="authorLabel">
     <property name="text">
      <string>Author:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="descriptionlabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Requirements:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="5">
    <widget class="QListWidget" name="pluginView">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QListWidget" name="requirementsView"/>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="showAllCheckBox">
     <property name="text">
      <string>Show internal plugins (always activated)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="installReqButton">
     <property name="text">
      <string>Install Requirements</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pluginView</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>Plugins</receiver>
   <slot>plugin_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>installReqButton</sender>
   <signal>clicked()</signal>
   <receiver>Plugins</receiver>
   <slot>install_req_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showAllCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>Plugins</receiver>
   <slot>show_all_toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pluginView</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>Plugins</receiver>
   <slot>activate_plugin_toggled(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>plugin_selected()</slot>
  <slot>install_req_clicked()</slot>
  <slot>show_all_toggled(bool)</slot>
  <slot>activate_plugin_toggled(QListWidgetItem*)</slot>
 </slots>
</ui>
